plugins/Eq/EqCurve.cpp

```diff
@@ -304,37 +319,18 @@ float EqHandle::getLowCutCurve( float x )
 	a2 /= a0;
 	a0 = 1;
 
-	double w;
-	double PHI;
-	double gain;
-	double freq = xPixelToFreq( x );
-	w = 2 * PI * freq / SR ;
-	PHI = pow( sin( w/2), 2 ) * 4;
-	gain = 10 * log10( pow( b0 + b1 + b2 , 2 )
-					  + ( b0 * b2 * PHI - ( b1 * ( b0 + b2 ) + 4 * b0 * b2 ) ) * PHI )
-					  - 10 * log10( pow( 1 + a1 + a2, 2 ) + ( 1 * a2 * PHI - ( a1 * ( 1 + a2 ) + 4 * 1 * a2 ) ) * PHI );
-
+	double freq = xPixelToFreq( x, m_width );
+	double gain = calculateGain( freq, a1, a2, b0, b1, b2 );
 	if ( m_hp24 )
 	{
-		gain = gain + 10 * log10( pow( b0 + b1 + b2 , 2 )
-						  + ( b0 * b2 * PHI - ( b1 * ( b0 + b2 ) + 4 * b0 * b2 ) ) * PHI )
-						  - 10 * log10( pow( 1 + a1 + a2, 2 ) + ( 1 * a2 * PHI - ( a1 * ( 1 + a2 ) + 4 * 1 * a2 ) ) * PHI );
-
+		gain = gain * 2;
 	}
-
 	if ( m_hp48 )
 	{
-		gain = gain + 10 * log10( pow( b0 + b1 + b2 , 2 )
-						  + ( b0 * b2 * PHI - ( b1 * ( b0 + b2 ) + 4 * b0 * b2 ) ) * PHI )
-						  - 10 * log10( pow( 1 + a1 + a2, 2 ) + ( 1 * a2 * PHI - ( a1 * ( 1 + a2 ) + 4 * 1 * a2 ) ) * PHI );
-
-		gain = gain + ( 10 * log10( pow( b0 + b1 + b2 , 2 )
-						  + ( b0 * b2 * PHI - ( b1 * ( b0 + b2 ) + 4 * b0 * b2 ) ) * PHI )
-						  - 10 * log10( pow( 1 + a1 + a2, 2 ) + ( 1 * a2 * PHI - ( a1 * ( 1 + a2 ) + 4 * 1 * a2 ) ) * PHI ));
-
+		gain = gain * 4;
```

Comment:
Are you sure it is `gain * 4`? Two sums means `gain * 3`. On the other side, regarding `m_hp48` 48 is twice 24 (`gain * 2`); are you changing it because of this?